app/controllers/api/v1/users_controller.rb

```diff
@@ -29,6 +29,51 @@ def update
         end
       end
 
+      def exist
+        userData = params[:user]
+        Rails.logger.info "************ exist is = #{userData[:email]}"
+        user = User.where(email: userData[:email]).first
```

Comment:
Since the `userData` is only being used once, you can save the extra variable assignment, and go straight to:

```ruby
user = User.where(email: params[:user][:email]).first
```

And a cool alternative to the `.where(...).first` pattern is the new'ish `.find_by`.  It either returns the singular record in question, or `nil`, if none is found.

```ruby
user = User.find_by(email: params[:user][:email])
```

This all assumes that we will **always** have a `:user` and `:email` populated in the params.

Reply:
done.